### ISSUE TITLE
logging-not-lazy doesn't handle .format

### ISSUE THREAD

**Issue by chdsbd** [PyCQA/pylint#2730]
### Steps to reproduce
```python
import logging

username = 'jdoe'
logging.info(f"{username} logged in")         # pylint complains
logging.info("{} logged in".format(username)) # no pylint complaint
logging.info("%s logged in", username)        # good case, pylint doesn't complain
```

### Current behavior
The following doesn't error like other non-lazy logging statements.
```python
logging.info("{} logged in".format(username))
```

### Expected behavior
The following should be detected by pylint as violating `logging-not-lazy`.
```python
logging.info("{} logged in".format(username))
```

### pylint --version output
```
pylint 2.2.2
astroid 2.1.0
Python 3.7.0 (default, Oct  2 2018, 09:20:07) 
[Clang 10.0.0 (clang-1000.11.45.2)]
```

This may be a regression of https://github.com/PyCQA/pylint/issues/317

**Comment by chdsbd** [comment]
I wanted `logging-format-interpolation`. I enabled that and it catches this error.
